frontend/src/metabase/home/containers/HomepageApp.jsx

```diff
@@ -4,7 +4,7 @@ import { connect } from "react-redux";
 import { t } from 'c-3po'
 import { push } from "react-router-redux";
 
-import Greeting from "metabase/lib/greeting";
+import { Greeting } from "metabase/lib/greeting";
```

Comment:
If we're not going to have Greeting be the default export I'd say we should probably just not bother with an object and have the various methods just be plain functions and do `import { sayHello } from 'metabase/lib/greeting' `

Reply:
Good idea. Or as we also want to export greetingPrefixes and other things we rename it to Greeter.

Reply:
Yeah I think plain functions is probably the way to go here.